server/src/resolvers/mutation/message.ts

```diff
@@ -42,4 +43,26 @@ export default {
     })
     return result
   },
+  async deleteMessage(parent, {messageId}, ctx: Context, info) : Promise<Message> {
+    const userId = getUserId(ctx)
+    const message : Message = await ctx.db.query.message({
+      where: {
+        id: messageId
+      }
+    }, 
+    `{
+       author {
+         id
+       }
+     }`)
+     if(!(message.author.id === userId)) throw new Error(`You aren't owner of this message`)
```

Comment:
Ja bym dał tutaj "You are not the author of this message!"

Reply:
Sound's better @Bjornskjald 👍